minio/minio.go

```diff
@@ -71,7 +72,7 @@ func (s *Storage) Get(key string) ([]byte, error) {
 	}
 
 	// convert to byte
-	buf := new(bytes.Buffer)
+	buf := new(bytebufferpool.ByteBuffer)
```

Comment:
This is not true usage. You can check https://github.com/valyala/bytebufferpool/blob/master/bytebuffer_example_test.go

Reply:
done

Reply:
Don't forget to it put it back with defer

Reply:
@mstgnz One last comment, and it should be good to go

Reply:
Added put it back with defer.